docs/managing-compute-resources.asciidoc

```diff
@@ -100,13 +100,13 @@ spec:
     type: Container
 ----
 
-With the above restriction in place, if you attempt to create an Elasticsearch object without defining the `resources` section, it will fail to start with an error similar to the following:
+With the above restriction in place, if you create an Elasticsearch object without defining the `resources` section, you will get the following error:
```

Comment:
It's not _exactly_ the same error but I guess I am splitting hairs here now :smile: